src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs

```diff
@@ -1685,6 +1685,7 @@ public C(int value)
             await VerifyCSharpAdditionalFileFixAsync(source, shippedText, unshippedText, fixedUnshippedText);
         }
 
+        [Fact]
```

Comment:
Nice!

Shall it be moved to a separate PR?

Reply:
No, this fixes the VSTHRD200 violation on the method.